corehq/apps/case_search/templates/case_search/csql_fixture_configuration.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'hqwebapp/bootstrap5/base_section.html' %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% js_entry "hqwebapp/js/htmx_and_alpine" %}
+
+{% block page_title %}
+  {% trans 'CSQL Fixture Configuration' %}
+{% endblock %}
+
+{% block page_content %}
+  {{ csql_fixture_configurations|json_script:"fixtureConfig" }}
+  <p>Please see <a href="https://dimagi.atlassian.net/wiki/spaces/USH/pages/2680389646/Module+Badges">the documentation</a> for this feature.</p>
+  <div x-data="{
+                validationFailed: false,
+                isDirty: false,
+                rows: JSON.parse(document.getElementById('fixtureConfig').textContent),
```

Comment:
That's cool, I hadn't seen `|json_script` before.  You may also be able do this in-line like so, but I'm not certain what the difference in escaping will be:
```suggestion
                rows: {% html_attr csql_fixture_configurations %},
```

Reply:
it's referenced a few lines above in the file - this is the code that pulls that into the attr.